src/utils/Utils.ts

```diff
@@ -21,11 +21,13 @@ export class Utils {
         );
       })
       .filter((record: any) => {
-        // Filter by testTypeClassification
+        // Filter by testTypeClassification or testTypeClassification, testResult and ivaDefects present and populated
         if (record.testTypes && record.testTypes.testTypeClassification) {
           return (
-            record.testTypes.testTypeClassification ===
-            "Annual With Certificate"
+            record.testTypes.testTypeClassification === "Annual With Certificate" ||
+            record.testTypes.testTypeClassification === "IVA With Certificate" &&
```

Comment:
I think this new set of conditions needs a bracket around the whole thing, otherwise they may get treated individually and the wrong test results may pass

Reply:
Happy to put a bracker around those, but did you have a particular scenario in mind that hasn't been covered by the unit tests?

I thought I had added a use case for each individual check to validate that it only gets through when all 3 are met.

Reply:
@m-coslett 

I have re-pushed with the parenthesis around the second group of logic.